src/main/java/racingcar/view/InputView.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.view;
+
+import racingcar.exception.ErrorMessage;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    public static int input(String content) {
+        System.out.println(content);
+        return validatedInput(SCANNER.nextInt());
+    }
+
+    private static int validatedInput(final int input) {
+        if (input < 0) {
+            throw new IllegalArgumentException(ErrorMessage.MORE_THAN_ZERO);
+        }
+        return input;
+    }
```

Comment:
```suggestion
    private static int validatedInt() {
        int input = SCANNER.nextInt();
        if (input < 0) {
            throw new IllegalArgumentException(ErrorMessage.MORE_THAN_ZERO);
        }
        return input;
    }
```
이런 방식은 어때요?

Reply:
이 방식도 validate 와 return 을 한 메서드에서 처리하게 되는 거 같아 다른 메서드로 분리해서 변경하려고 합니다! 

```suggestion
    private static int input(String content) {
        System.out.println(content);
        int value = SCANNER.nextInt();
        validateInput(value);

        return value;
    }

    private static void validateInput(final int input) {
        if (input <= 0) {
            throw new IllegalArgumentException(ErrorMessage.MORE_THAN_ZERO);
        }
    }
```